### ISSUE TITLE
Replace black arrow (`▶`) with `►` 

### ISSUE THREAD

**Issue by tuanchauict** [tuanchauict/MonoSketch#605]
Not sure its related, but I would love to have the arrows like asciiflow.
`This ►  instead of ▶`

It makes the vertical lines misaligned

You can notice it in the example on the README.md page and below.

Example :
Current :
```
                 ─       │           ├───────○──┤/  547 │                    
                GND      │           │       │  │ ▶     │                    
                         │           │      ╭┴╮   │     │                    
```

With suggested arrow :
```
                 ─       │           ├───────○──┤/  547 │                    
                GND      │           │       │  │ ►     │                    
                         │           │      ╭┴╮   │     │                    
                         
``` 

I like the idea that we can keep it as vanilla as possible without having to use fonts like jetbrains mono (even if its a cool font), but can use fonts like monospace or courier.

_Originally posted by @klausdk in https://github.com/tuanchauict/MonoSketch/issues/586#issuecomment-2144619182_

**Comment by tuanchauict** [comment]
Answer: https://github.com/tuanchauict/MonoSketch/issues/586#issuecomment-2151069590

**Comment by tuanchauict** [comment]
@klausdk I create a new issue for your request.
This is being addressed with #604

**Comment by tuanchauict** [comment]
On alpha now
<img width="139" alt="image" src="https://github.com/tuanchauict/MonoSketch/assets/955306/977d5733-b545-4c0a-a2b5-629bbffd4949">
